examples_utils/benchmarks/platform_assessment/assess_platform.py

```diff
@@ -4,9 +4,10 @@
 import logging
 import subprocess
 import sys
-import yaml
 from pathlib import Path
 
+import yaml
```

Comment:
why? Is this done by some import linter?

Reply:
Can revert, this was done by isort

Reply:
Yes please, isort has some odd corners in its policies that I havent looked into but I see no benefit to having the yaml import on its own line aside from the others.